internal/metrics/metrics.go

```diff
@@ -215,7 +215,7 @@ func (svc *Service) Start(stop <-chan struct{}) error {
 }
 
 func registerHealthCheck(mux *http.ServeMux) {
-	mux.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
We should enable both `/healthz` and `/health` to give us a way to deprecate the old path.

Reply:
SGTM. Please add a TODO and point it to an issue to remove the old endpoint in 0.14 or 0.15

Reply:
Done